### PR TITLE
Added TAC7316C

### PR DESCRIPTION
**Pull request by markwv**
Added Tuya-based 3-phase power meter with relay. 

Possible improvements:
- This device updates every two minutes, not sure there is a way to override this.
- Image of the device

### CODE REVIEW

File: devices/tuya.js [hunk 1]
```diff
@@ -4351,4 +4351,40 @@ module.exports = [
...
+        model: 'TAC7361C BI 3-phase power meter',
+        vendor: 'TuYa',
+        description: '3-phase clamp power meter with relay',
+        extend: tuya.extend.switch(),
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        configure: tuya.configureMagicPacket,
+        whiteLabel: [{vendor: 'Wenzhou Taiye Electric', model: 'TAC7361C BI'}],
+        exposes: [
+            exposes.binary('state', ea.STATE_SET, 'ON', 'OFF').withDescription('Turn the relay ON/OFF'),
```
Comment (Koenkk) [inline devices/tuya.js:4365]: Please use `e.switch().setAccess('state', ea.STATE_SET)` here

File: devices/tuya.js [hunk 2]
```diff
@@ -4351,4 +4351,40 @@ module.exports = [
...
+            tuya.exposes.powerWithPhase('a'),
+            tuya.exposes.powerWithPhase('b'),
+            tuya.exposes.powerWithPhase('c'),
+            tuya.exposes.currentWithPhase('a'),
+            tuya.exposes.currentWithPhase('b'),
+            tuya.exposes.currentWithPhase('c'),
+        ],
+        meta: {tuyaDatapoints: [
+            [16, 'state', tuya.valueConverter.onOff],
+            [1, 'consumed_energy', tuya.valueConverter.divideBy100],
```
Comment (Koenkk) [inline devices/tuya.js:4381]: Please rename to `energy`

**Comment by markwv** [comment]
I have processed your comments, see the commit above.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**